src/main/webapp/WEB-INF/tags/student/feedbackSubmissionEdit/submitButton.tag

```diff
@@ -0,0 +1,22 @@
+<%@ tag description="feedbackSubmissionForm.tag - Submit button" %>
+<%@ tag import="teammates.common.util.Const"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="preview" type="java.lang.Boolean" required="true" %>
+<%@ attribute name="submittable" type="java.lang.Boolean" required="true" %>
+
+<c:set var="TOOLTIP_EDIT_SAVE" value="<%=Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE%>"/>
```

Comment:
Is there a reason for explicitly setting this?
It seems fine just using the constant directly...